docs/en/roadmap/template.md

```diff
@@ -0,0 +1,39 @@
+# Roadmap
+
+Date: 2021-01-01 to 2021-03-30
+
+> Note: add roadmap entry to to `roadmap/README.md`
+
+## Core Platform
+
+- K8s controllers
+- Core workloads/traits
+- Server components
+- Architecture and model
+
+## User Experience
+
+- Devtools: CLI/UI/Appfile
+- SDK/Framework
+
+## Deployment and Operations
+
+- Diagnostics and debugging
+
+## Third-party integrations
+
+- CI/CD, GitOps
+- Application development framework
+- Third party workloads/traits
+
+## Testing
+
+- Test infrastructure
+  - CI (Github Actions) and hosts
+  - codecov
+- Unit/e2e test scripts and go code
```

Comment:
How about we simply use `Unit/e2e test cases`? I got confused the first time I read `go code`. Now I realized `scripts` and `go code` are used to write unit/e2e test cases.

Reply:
Good idea!
Updated.